src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -2271,6 +2271,25 @@ public static Vector64<sbyte> Shuffle(Vector64<sbyte> vector, Vector64<sbyte> in
             return result;
         }
 
+        /// <summary>Creates a new vector by selecting values from an input vector using a set of indices.
+        /// Behavior is platform-dependent for out-of-range indices.</summary>
+        /// <param name="vector">The input vector from which values are selected.</param>
+        /// <param name="indices">The per-element indices used to select a value from <paramref name="vector" />.</param>
+        /// <returns>A new vector containing the values from <paramref name="vector" /> selected by the given <paramref name="indices" />.</returns>
+        /// <remarks>Unlike Shuffle, this method delegates to the underlying hardware intrinsic without ensuring that <paramref name="indices"/> are normalized to [0, 7].
+        /// On hardware with <see cref="AdvSimd"/> support, this method behaves the same as Shuffle.</remarks>
```

Comment:
```suggestion
        /// <remarks>Unlike Shuffle, this method delegates to the underlying hardware intrinsic without ensuring that <paramref name="indices"/> are normalized to [0, 7].</remarks>
```

It's always the same as `Shuffle`, so I don't think we need to call out `AdvSimd`.